quickstart/101-vm-with-infrastructure/main.tf

```diff
@@ -111,8 +111,8 @@ resource "azurerm_linux_virtual_machine" "my_terraform_vm" {
 
   source_image_reference {
     publisher = "Canonical"
-    offer     = "UbuntuServer"
-    sku       = "18.04-LTS"
+    offer     = "0001-com-ubuntu-server-jammy"
```

Comment:
I think the feedback was more around ensuring that the VM size in the examples are still current and supported rather than the image. I don't think there is necessarily an issue with updating the image to a more current one but I would align with the portal which defaults to Ubuntu 20.04 LTS.

Reply:
Understood, but when I asked Cynthia Nottingham (who manages the Compute docs), she recommended making these changes.

Reply:
Okey-dokey.